<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>高光谱图像分类系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>851</width>
      <height>541</height>
     </rect>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>857</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件&amp;F</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
    <addaction name="separator"/>
    <addaction name="action_6"/>
    <addaction name="separator"/>
    <addaction name="action_8"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑&amp;E</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_S">
    <property name="title">
     <string>设置&amp;S</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>训练&amp;T</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>分类&amp;C</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助&amp;H</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_L">
    <property name="title">
     <string>视图&amp;V</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_E"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_L"/>
   <addaction name="menu_S"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Open">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>保存分类结果</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>打印</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
